### ISSUE TITLE
Search filed with nested value

### ISSUE THREAD

**Issue by fbmfbm** [HC200ok/vue3-easy-data-table#142]
Hi,
i'm looking to use search input for field with nested value.

It's working well if field  is value but don't work on nested field : 

ex : 
 const searchField = ref("reference");
 const searchValue = ref();

        const headers = [
            { text: "Réf.", value: "reference" },
            { text: "Status", value: "status" },
            { text: "Client", value: "user.email" },
        ]

passing 'reference' or 'staus' for searchField is working but 'user.email' don't !

How can we search nested field ?

Thank's

**Comment by HC200ok** [comment]
@fbmfbm Please update to the latest version `1.5.9`

**Comment by fbmfbm** [comment]
@HC200ok thank's for your answer,

i'm switching to 1.5.9 but for now it's not working...maybe i'm in trouble : i have the searchField referenced to "user.email" and change value for searchValue to some user.email text and nothing is returned. 
I'm in Vue3 composition Api...can this make some difference ? 
(other field sarching working well).

**Comment by HC200ok** [comment]
@fbmfbm Please show me the example of your items data.

**Comment by fbmfbm** [comment]
array of rows lyke this : 

`
{
"id":"16",
"reference":"test_b1cN7",
"status":"paid",
"tax_total":8356,
"created_at":"2022-10-21 14:40:20",
"delivery_total":0,
"order_total":50136,
"user":
{"id":"15",
"name":"myName",
"email":"test@gg.com",
}
`

**Comment by HC200ok** [comment]
@fbmfbm 
```vue
<template>
  <span>search field:</span>
  <select v-model="searchField">
    <option>name</option>
    <option>user.email</option>
  </select>
  <DataTable
      :headers="headers"
      :items="items"
      :search-field="searchField"
      :search-value="searchValue"
    />
</template>

<script lang="ts" setup>
const searchField = ref('user.email');
const searchValue = ref('');
</script>
```
![chrome-capture-2022-9-31](https://user-images.githubusercontent.com/11552575/198941126-67981e4e-e27f-4390-8825-154971bb9854.gif)

it works well in my local, Is it possible you are using the searching feature with the filtering feature that leads to no result?

**Comment by fbmfbm** [comment]
Humm,
I'm doing lyke you but for me it's not working...no filter. 


I'm create a codesandbox online exemple with simple data : https://codesandbox.io/s/relaxed-wind-mmc8nz?file=/src/components/OrdersTable.vue and i invite you with our email.

**Comment by HC200ok** [comment]
@fbmfbm I just published a new version `1.5.11` and then it works, maybe there was something wrong with the publishment of `1.5.9` and `1.5.10`. Sorry for that...

**Comment by fbmfbm** [comment]
Yep it's working !
Thank's for you reactivity. 
Good job man !
